projects/react/src/button/__snapshots__/index.test.tsx.snap

```diff
@@ -18,7 +18,6 @@ exports[`CdsButton snapshot 1`] = `
     danger
   </cds-button>
   <cds-button
-    disabled="true"
```

Comment:
This appears as if the button was disabled before and now isn't, but it's actually fine. I think it's a slight change in how Lit works, but it still works correctly. The snapshot tests are almost useless, they're a holdover from the older enzyme tests, but are nice for occasionally catching a change.